src/ontology/uberon-edit.obo

```diff
@@ -222238,6 +222238,17 @@ synonym: "fifth molar tooth" EXACT []
 is_a: UBERON:0003655 ! molar tooth
 relationship: dc-contributor https://orcid.org/0000-0003-2699-3066
 
+[Term]
+id: UBERON:7500127
+name: otion
+def: "The most lateral anatomical point on the mastoid process of temporal bone." [http://desinfos.eu/archeozoologie.desinfos.eu/PERSO/Aides%20determination/Osteometrie/Von%20Den%20Driesch\,%201976\,%20A%20guide%20to%20the%20measurement%20of%20animal%20bones%20from%20archaeological%20sites.pdf, ISBN:9780873659505]
```

Comment:
The dbxref uses the term "mastoid region", which was in the originally submitted definition here. While substituting "mastoid process of temporal bone" seems reasonable, I cannot confirm this is the same based on the reference.